roles/openshift_master_certificates/tasks/main.yml

```diff
@@ -10,6 +10,7 @@
     src: "{{ openshift_master_config_dir }}/{{ item.1 }}"
     dest: "{{ openshift_generated_configs_dir }}/{{ item.0.master_cert_subdir }}/{{ item.1 }}"
     state: hard
+    force: true
```

Comment:
Technically, this is forcing a hardlink, not a symlink :grinning: 

That said, I don't see any issue with this. I'm assuming the error crops up when they are attempting to install after having generated_configs_dir already populated with some previous certs.